SpatialGDK/Source/SpatialGDKEditor/Private/EditorExtension/GridLBStrategyEditorExtension.cpp

```diff
@@ -0,0 +1,25 @@
+#include "GridLBStrategyEditorExtension.h"
+#include "SpatialGDKEditorSettings.h"
+
+class UGridBasedLBStrategy_Spy : public UGridBasedLBStrategy
+{
+public:
+	using UGridBasedLBStrategy::WorldWidth;
+	using UGridBasedLBStrategy::WorldHeight;
+	using UGridBasedLBStrategy::Rows;
+	using UGridBasedLBStrategy::Cols;
+};
+
+bool FGridLBStrategyEditorExtension::GetDefaultLaunchConfiguration(const UGridBasedLBStrategy* Strategy, FWorkerTypeLaunchSection& OutConfiguration, FIntPoint& OutWorldDimensions) const
+{
+	const UGridBasedLBStrategy_Spy* StrategySpy = static_cast<const UGridBasedLBStrategy_Spy*>(Strategy);
+
+	OutConfiguration.Rows = StrategySpy->Rows;
+	OutConfiguration.Columns = StrategySpy->Cols;
+	OutConfiguration.NumEditorInstances = StrategySpy->Rows * StrategySpy->Cols;
+
+	// m ? cm ?
```

Comment:
LB strategy dimensions are in cm, launch config is in metres, this sucks and should probably be made consistent but maybe out of scope for fixing here